packages/serverless-component/serverless.js

```diff
@@ -71,6 +71,9 @@ class NextjsComponent extends Component {
     const nextConfigPath = inputs.nextConfigDir
       ? path.resolve(inputs.nextConfigDir)
       : process.cwd();
+    const nextStaticPath = inputs.nextStaticDir || "";
+    const staticPath = nextStaticPath || nextConfigPath;
+    const cloudFrontConfigs = inputs.cloudfront || {};
```

Comment:
Could you rename the `cloudFrontConfigs` var. to `customCloudFrontConfig`, just to make clearer the intent.